server/api_server/api_server_start.py

```
@@ -1930,6 +1932,33 @@ def check_auth(payload=None):
+# Health endpoint
+# --------------------------
+@app.route("/health", methods=["GET"])
+@validate_request(
+    operation_id="check_health",
+    summary="System Health Check",
+    description="Retrieve system vitality metrics including database size, memory pressure, system load, disk usage, and CPU temperature.",
+    response_model=HealthCheckResponse,
+    tags=["system", "health"],
+    auth_callable=is_authorized
+)
+def check_health(payload=None):
```

Ruff (0.15.1)

[warning] Unused function argument: `payload`

(ARG001)